tests/univ3/test_position_creation.py

```diff
@@ -0,0 +1,54 @@
+from math import sqrt, log
+
+from brownie import interface
+
+from great_ape_safe.ape_api.helpers.uni_v3.uni_v3_sdk import BASE, Q96
+
+# misc.
+FEE_TIER = 500
+PRICE_1 = 1
+PRICE_101 = 1.01
+
+
+def test_position_creation(treasury, liq, wbtc):
+    decimals_diff = liq.decimals() - wbtc.decimals()
+
+    treasury.init_uni_v3()
+
+    pool_address = treasury.uni_v3.factory.createPool(liq, wbtc, FEE_TIER).return_value
+    pool = interface.IUniswapV3Pool(pool_address, owner=treasury.account)
+
+    sqrt_of_p = sqrt(1e18 / 10 ** wbtc.decimals())
+    sqrt_price_x_96 = sqrt_of_p * Q96
+
+    pool.initialize(sqrt_price_x_96)
+
+    token_0_init_seeding_amount = 2.5e8
+    token_1_init_seeding_amount = 2.5e18
+
+    token_id = treasury.uni_v3.mint_position(
+        pool,
+        PRICE_1,
+        PRICE_101,
+        token_0_init_seeding_amount,
+        token_1_init_seeding_amount,
+    )
+
+    positions = treasury.uni_v3.nonfungible_position_manager.positions(token_id)
+
+    # TODO: rounding to 4 decimal places, values are not matching the expected?
+    price_lower_ticket = round(
```

Comment:
Minor typo in here. Mean ticker or tick, right?

Reply:
good catch, fixed in 9d360d763ed5c69b7e25fb393919e271bf74f4da